packages/fxa-content-server/app/styles/tailwind/ctas.css

```diff
@@ -4,9 +4,6 @@
 
 /* Max-height is a likely temp "hack" for .spinner until it's converted to TW */
 /* font-color is also a hack until all buttons are TWified */
```

Comment:
Will you move these comments over as well? Not sure if font-color is applicable though since I don't see the class but the max-height comment for sure.

Reply:
OH yes I will! Thanks!